### PR TITLE
:whale: setting up a docker environment for react

### PR DESCRIPTION
**Pull request by paparazzie**
@naj21 I initialized the application again with **git flow init**, so it created a new develop branch, you can discard the old dev branch, I also changed the directory ish to app.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Installation
+### Installlation without docker
```
Comment (hackthievist) [inline README.md:2]:
```suggestion
### Installation without docker
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+* Clone the repository
+* make sure that Node.js is installed
+* cd into public-idea-list
```
Comment (hackthievist) [inline README.md:5]: Is this still valid? I thought Jessica changed the top level folder name to `app`

Comment (paparazzie) [reply README.md:5]: oh shit, I missed that, I didn't update the README, to add 'app'. I would do that now

**Comment by naj21** [comment]
Don't merge to master yet, when we are done with the setups we'll create a release and merge to master.

**Pull request closed without merging**